docs/en/test_cards/mc_06_optical_flow.md

```diff
@@ -38,6 +39,16 @@ Ensure that the drone can go into Altitude / Position flight mode while still on
 
 &nbsp;&nbsp;&nbsp;&nbsp;❏ Pitch/Roll/Yaw response set to pitch/roll/yaw rates
 
+❏ Varying height terrain
+
+&nbsp;&nbsp;&nbsp;&nbsp;❏ But boxes on the ground to create varying heights in terrain
+
+&nbsp;&nbsp;&nbsp;&nbsp;❏ Take off in position mode and fly over the boxes such that the downward facing rangefinder varies in value
+
+&nbsp;&nbsp;&nbsp;&nbsp;❏ Do a few passes with varying amounts of time over the boxes (1-30 seconds if possible)
+
+&nbsp;&nbsp;&nbsp;&nbsp;❏ Drone should not raise in height when flying over boxes
+
```

Comment:
but make sure cs_rng_hgt status flag is true... make sure you're within the aid src velocity threshold

Reply:
yeah

Reply:
added